libs/ha/ha_comp_libs.py

```diff
@@ -214,22 +226,22 @@ def get_ha_log_prop(node_obj, log_name: str, kvalue: int, fault_tolerance: bool
 
     @staticmethod
     def get_ha_log_wc(node_obj, log_index: int):
-        '''
+        """
         Helper function to get word count of a file
         :param node_obj: Master node(Logical Node object)
         :param log_index: name of the log
         0 - k8s | 1 - fault_tolerance | 2 - health_monitor
         :return: wc_count(Word count of a ha log file)
-        '''
+        """
         pvc_list = node_obj.execute_cmd(common_cmd.HA_LOG_PVC, read_lines=True)
-        hapvc = None
+        hapvc = wc_count = None
         for hapvc in pvc_list:
             if common_const.HA_POD_NAME_PREFIX in hapvc:
                 hapvc = hapvc.replace("\n", "")
                 LOGGER.info("hapvc list %s", hapvc)
                 break
-        wc_count_cmd = common_const.HA_LOG + hapvc + "/log/ha/*/" + \
-                          common_const.HA_SHUTDOWN_LOGS[log_index]
+        wc_count_cmd = common_const.HA_LOG + hapvc + "/log/ha/*/" + common_const.HA_SHUTDOWN_LOGS[
+            log_index]
```

Comment:
what was error here? the chnaged one looks a bit not good instead

Reply:
> what was error here? the chnaged one looks a bit not good instead

Minor with indent issue.